packages/contracts/scripts/utils/deployVerifiersAndHelpers.ts

```diff
@@ -16,15 +15,12 @@ export async function deploySingleContract(
 ): Promise<ethers.Contract> {
     try {
         // Create a new ContractFactory instance
-        const _ContractFactory = new ethers.ContractFactory(
+        const ContractFactory = new ethers.ContractFactory(
             abi,
             bytecode,
             deployer
         )
 
-        // Apply global factory modifications
-        const ContractFactory = await globalFactory(_ContractFactory)
```

Comment:
I thought the reason we are using `globalFactory` is due to [create2](https://socket.dev/npm/package/global-factory), which is able to generate the same address. But if the deployer would be hard to maintain, then this is good to me